pkg/strategy/support/strategy.go

```diff
@@ -439,7 +390,48 @@ func (s *Strategy) Run(ctx context.Context, orderExecutor bbgo.OrderExecutor, se
 	s.orderExecutor = orderExecutor
 
 	// StrategyController
-	s.status = types.StrategyStatusRunning
+	s.Status = types.StrategyStatusRunning
+
+	s.OnSuspend(func() error {
+		var err error = nil
+		// Cancel all order
+		for _, order := range s.orderStore.Orders() {
```

Comment:
orderStore includes closed orders, I think you need to check the open orders~

Reply:
is there a function to check this, or a function to remove filled and cancelled orders in orderStore?

Reply:
Cancelled orders (and not filled) are automatically removed from the order store once you bind it to the stream.

OrderStore is used for tracking order ids for the trades.

You need to call ex.QueryOpenOrders() to get these pending orders (orders still on the book)

Reply:
In the bollmaker strategy, we use activeMakerOrders to track these open orders:

	s.activeMakerOrders = bbgo.NewLocalActiveOrderBook(s.Symbol)
	s.activeMakerOrders.BindStream(session.UserDataStream)

Reply:
use LocalActiveOrderBook to achieve this